src/GeometricFigures/Circle.java

```diff
@@ -0,0 +1,17 @@
+package GeometricFigures;
+
+public class Circle implements Geometric{
+    double diameter;
+    public Circle(double diameter) {
+        this.diameter = diameter;
+    }
+    public double getArea() {
+        return Math.PI * Math.pow(diameter / 2, 2);
```

Comment:
consider using radius instead of diameter

Reply:
Is this really have any importance?